release/release_logs/1.13.0rc0/microbenchmark.json

```diff
@@ -0,0 +1,274 @@
+{
+    "1_1_actor_calls_async": [
+        6411.321046340747,
+        144.09700737494842
+    ],
+    "1_1_actor_calls_concurrent": [
+        4762.298226184261,
+        258.8355744733599
+    ],
+    "1_1_actor_calls_sync": [
+        2481.9516295447324,
+        47.1459294358713
+    ],
+    "1_1_async_actor_calls_async": [
+        3655.8947859012255,
+        287.72739922703295
+    ],
+    "1_1_async_actor_calls_sync": [
+        1760.6933323973485,
+        29.402298751865963
+    ],
+    "1_1_async_actor_calls_with_args_async": [
+        2384.670664299169,
+        152.82405396732784
+    ],
+    "1_n_actor_calls_async": [
+        10671.14624877688,
+        526.5357298149904
+    ],
+    "1_n_async_actor_calls_async": [
+        11120.569744924334,
+        54.99325174877086
+    ],
+    "client__1_1_actor_calls_async": [
+        917.4274099113774,
+        7.679634468965718
+    ],
+    "client__1_1_actor_calls_concurrent": [
+        919.5579846613139,
+        10.193898071436422
+    ],
+    "client__1_1_actor_calls_sync": [
+        531.1359243446507,
+        2.1392467853878676
+    ],
+    "client__get_calls": [
+        1149.095406633416,
+        6.839805811255162
+    ],
+    "client__put_calls": [
+        848.653367638983,
+        22.560178124190337
+    ],
+    "client__put_gigabytes": [
+        0.044837209794706845,
+        0.0007840397458485403
+    ],
+    "client__tasks_and_get_batch": [
+        0.8172203467773957,
+        0.020469333663094173
+    ],
+    "client__tasks_and_put_batch": [
+        8792.749660258994,
+        19.10705960496608
+    ],
+    "multi_client_put_calls_Plasma_Store": [
+        9745.670014416988,
+        126.20971804569032
+    ],
+    "multi_client_put_gigabytes": [
+        36.85514798847486,
+        1.6722498011347091
+    ],
+    "multi_client_tasks_async": [
+        37237.079715148226,
+        777.4796877284789
+    ],
+    "n_n_actor_calls_async": [
+        34770.47668215435,
+        842.1881338319646
+    ],
+    "n_n_actor_calls_with_arg_async": [
+        2634.652949167727,
```

Comment:
Same for this 3336 -> 2634

Reply:
Based on the history, I feel this is false alarm as well.